src/ai/defaultai_warfare.cc

```diff
@@ -345,7 +345,7 @@ bool DefaultAI::check_enemy_sites(const Time& gametime) {
                            -3 :
                            0;
 				inputs[10] = (observer.second.mines_nearby == ExtendedBool::kTrue) ? 3 : -1;
-				inputs[11] = (observer.second.mines_nearby == ExtendedBool::kTrue) ? 0 : 0;
+				inputs[11] = (observer.second.mines_nearby == ExtendedBool::kTrue) ? 2 : 0;
```

Comment:
This is no really a clang_tidy change :-)

Reply:
> 
> 
> This is no really a clang_tidy change :-)

it is as it was found by clang-tidy.

Reply:
Well before it was the same as 
`inputs[11] = 0`
now you changed the values to 2 and 0, that is a semantic change

Reply:
yes it was a semantic error. this was correctly found by clang-tidy checking for redundant expressions. So this should stay in here